hypercertEvaluation.json

```diff
@@ -0,0 +1,42 @@
+{
+  "lexicon": 1,
+  "id": "app.certified.hypercert.evaluation",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "An evaluation of a hypercert or other claim",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["subject", "evaluatorDID", "summary", "createdAt"],
+        "properties": {
+          "subject": {
+            "type": "ref",
+            "ref": "com.atproto.repo.strongRef",
+            "description": "Reference to what is being evaluated (hypercert, measurement, etc.)"
+          },
+          "evaluatorDID": {
+            "type": "string",
```

Comment:
Similar question here - does it make sense to allow multiple people to contribute to an evaluation?  Feels like that could be quite common.  We could just handle that by using an org DID, but that seems imprecise.